### PR TITLE
Remove Scene dependency from the TextServer, use scalable hex box font instead of built-in bitmap one.

### PR DESCRIPTION
**Pull request by bruvzg**
Part of #53295

Removes Scene dependency from the TextServer (was used to load hex box font texture).
Instead of textures, generate hex code glyphs from the code (equivalent of 7-segment display text), also make it scalable.

Before:

Screenshot 2021-10-02 at 23 53 09

After:

Screenshot 2021-10-02 at 23 49 04

### CODE REVIEW

File: thirdparty/README.md [hunk 1]
```diff
@@ -138,11 +138,6 @@ Files extracted from upstream source:
   * Upstream: https://fonts.google.com/specimen/Open+Sans
   * Version: 1.10 (downloaded from Google Fonts in February 2021)
   * License: Apache 2.0
-- `Tamsyn*.png`:
-  * Upstream: http://www.fial.com/~scott/tamsyn-font/
-  * Version: 1.11 (2015)
-  * License: Tamsyn
```
Comment (akien-mga) [inline thirdparty/README.md:144]: You can also remove the file reference and license text from COPYRIGHT.txt.

**Comment by akien-mga** [comment]
The "afterx screenshot lacks the `0 0` hex box, is that expected?

**Comment by bruvzg** [comment]
> The "after" screenshot lacks the 0 0 hex box, is that expected?

`0x00` is a bug from the old version of the engine, and should be fixed in the current master.

File: thirdparty/README.md (hunk 1 above)
Comment (bruvzg) [reply thirdparty/README.md:144]: Done.

**Review by akien-mga — APPROVED** [review]
Looks great! Feels cleaner too as it's well encapsulated now.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**